### ISSUE TITLE
prefer-lodash-method shouldn't complain about Object.create(null)

### ISSUE THREAD

**Issue by lukeapage** [wix/eslint-plugin-lodash#94]
There is no equivalent in lodash for `Object.create(null)`. Calling `_.create(null)` doesn't do the same thing, so its misleading to suggest to use the lodash method

**Comment by mik01aj** [comment]
Hmm, how about `_.stubObject()`?

**Comment by ganimomer** [comment]
What is the difference between them?

**Comment by lukeapage** [comment]
```
_.create(null).hasOwnProperty
```
`hasOwnProperty() { [native code] }`
```
Object.create(null).hasOwnProperty
```
`undefined`

Lodash just defaults to the object prototype, creating an object that inherits Object methods onto the prototype, one creates a real plain object that has no prototype, meaning it is a true empty map that can be used like a map
